app/api/images/images.routes.js

```diff
@@ -71,7 +71,13 @@ module.exports = () => {
   router.get("/images/:id/georeferencers",
     validateDocumentedRequestParametersFor('GET', '/images/{id}/georeferencers'),
     controller.getGeoreferencers
-  );  
+  );
+
+  // Retrieve all georeferencers of an image.
```

Comment:
So it will not retrieve all georeferencers. It will findOne ??

Reply:
Bad copy/paste ( I update the comment but I think this comments are not necessary because we have openapi file).

In the end, find one because we need to know if there is a waiting_validation geolocalisation for a specific image